collections/impact-observatory-lulc-map.yaml

```diff
@@ -7,7 +7,7 @@ Description: |
 AdditionalInfoExternal:
     Title: Additional info
     Path: impact-observatory-lulc-map/README.MD 
-Image: impact-observatory-lulc-map/io-lulc-map.png
+Image: impact-observatory-lulc-map/io-lulc-ganges-delta.png
```

Comment:
On more last thing Chung, sorry for taking you back,

Could you create the image without labels , that would be perfect  

otheriwise everything works okay now